src/components/querycompt.tsx

```diff
@@ -54,12 +57,14 @@ QueryComponentProps) => {
        <Checkbox label=""defaultChecked={checkBoxChecked} onChange={checked} />
        <ColorPickerButton disabled={!checkBoxChecked} onColorPick={colorChanged} initialColor={color} />
          <Button styleType='high-visibility' onClick={remove}>Remove</Button>
-         <Textarea
+         <LabeledTextarea
              id='text-area'
              value={value}
              onChange={queryChanged}
              style={{ width: '100%' }}
-            disabled={!checkBoxChecked}
+             label=''
+             message={props.errormessage}
+             status='negative'
```

Comment:
The status is always negative, it should be positive if the query works and undefined if the query has never been run